lms/templates/registration/password_reset_email.html

```diff
@@ -1,21 +1,353 @@
+                                                        </tr>
+                                                        <tr style="border-collapse:collapse">
+                                                            <td align="center" style="padding:20px;Margin:0;font-size:0">
+                                                                <table border="0" width="100%" height="100%" cellpadding="0" cellspacing="0" role="presentation" style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px">
+                                                                    <tr style="border-collapse:collapse">
+                                                                        <td style="padding:0;Margin:0;border-bottom:1px solid #CCCCCC;background:none;height:1px;width:100%;margin:0px"></td>
+                                                                    </tr>
+                                                                </table></td>
+                                                        </tr>
+                                                        <tr style="border-collapse:collapse">
+                                                            <td align="center" style="padding:0;Margin:0;padding-bottom:20px;font-size:0">
+                                                                <table cellpadding="0" cellspacing="0" class="es-table-not-adapt es-social" role="presentation" style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px">
+                                                                    <tr style="border-collapse:collapse">
+                                                                        <td align="center" valign="top" style="padding:0;Margin:0;padding-right:30px"><a target="_blank" href="https://www.facebook.com/NAVOICA" style="-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;font-family:'open sans', 'helvetica neue', helvetica, arial, sans-serif;font-size:14px;text-decoration:underline;color:#5B5B5B"><img title="Navoica Facebook" src="${settings.LMS_ROOT_URL}/static/images/facebook-square-icon.png" alt={% trans "Facebook - Navoica page on Facebook"%} width="24" height="24" style="display:block;border:0;outline:none;text-decoration:none;-ms-interpolation-mode:bicubic"></a></td>
```

Comment:
w pliku są w użyciu settings do urli, ale brak importu -> from django.conf import settings

Reply:
Przede wszystkim to jest django_template, a nie mako_template.
Więc takie rzeczy: ${settings.LMS_ROT_URL} nie przejdą. Należy zmienne przekazywać przez context, a w przypadku staticów można zastosować taki zabieg: https://docs.djangoproject.com/en/1.11/howto/static-files/

Reply:
@krystianlygas proszę sprawdź czy jest ok. zrobiłam tak:: "{{ protocol }}://{{ domain }}{% static 'images/navoica-logo-email.png' %}"